test/test_datasources.py

```diff
@@ -35,7 +34,7 @@
     os.path.dirname(os.path.realpath(__file__)), "data", "test_spatialite.sqlite"
 )
 
-netcdf_groundwater_datasource_nc_path = os.path.join(
+results_3di_path = os.path.join(
```

Comment:
`threedi_results_path` is taalkundig netter. Volgorde.

En: hoofdletters van maken?